### PR TITLE
remove unused averages for 1D tagging

### PR DESCRIPTION
**Pull request by nicolasaunai**

### CODE REVIEW

File: src/amr/tagging/default_hybrid_tagger_strategy.h [hunk 1]
```diff
@@ -43,17 +43,11 @@ void DefaultHybridTaggerStrategy<HybridModel>::tag(HybridModel& model,
     {
         for (auto iCell = 0u, ix = start; iCell <= endCell; ++ix, ++iCell)
         {
-            auto Byavgm2 = 0.2 * (By(ix - 4) + By(ix - 3) + By(ix - 2) + By(ix - 1) + By(ix));
-            auto Byavgm1 = 0.2 * (By(ix - 3) + By(ix - 2) + By(ix - 1) + By(ix) + By(ix + 1));
             auto Byavg   = 0.2 * (By(ix - 2) + By(ix - 1) + By(ix) + By(ix + 1) + By(ix + 2));
             auto Byavgp1 = 0.2 * (By(ix - 1) + By(ix) + By(ix + 1) + By(ix + 2) + By(ix + 3));
```
Comment (PhilipDeegan) [inline src/amr/tagging/default_hybrid_tagger_strategy.h:47]: is the (ix + 3) not a bit suspect for interp1? (ie. two ghosts)

Comment (PhilipDeegan) [reply src/amr/tagging/default_hybrid_tagger_strategy.h:47]: looks like https://hephaistos.lpp.polytechnique.fr/teamcity/buildConfiguration/Phare_Phare_BuildGithubPrClang/71213?buildTab=log&focusLine=4776&linesState=1063

Comment (nicolasaunai) [reply src/amr/tagging/default_hybrid_tagger_strategy.h:47]: you are right. We should probably not take 5 points when on patch borders

Comment (nicolasaunai) [reply src/amr/tagging/default_hybrid_tagger_strategy.h]: Next commit should address that issue

**Comment by PhilipDeegan** [comment]
included in https://github.com/PHAREHUB/PHARE/pull/614

**Pull request closed without merging**